### PR TITLE
feature(devops): qa workflow

### PR DESCRIPTION
**Pull request by rafaiovlabs**

### CODE REVIEW

File: .github/workflows/qa.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: "UI deploy QA"
+on:
+  release:
```
Comment (pmprete) [inline .github/workflows/qa.yaml:3]: As we want this on merge to master it should be
```
on:
  push:
    branches:
      - master
```

**Review by pmprete — COMMENTED** [review]
Except my comment looks good

File: .github/workflows/qa.yaml (hunk 1 above)
Comment (rafaiovlabs) [reply .github/workflows/qa.yaml]: Done

**Review by pmprete — APPROVED** [review]
Awsome! Good work

**Pull request merged**